keras_cv/bounding_box/mask_invalid_detections.py

```diff
@@ -61,6 +61,7 @@ def mask_invalid_detections(bounding_boxes, output_ragged=False):
 
     boxes = bounding_boxes.get("boxes")
     classes = bounding_boxes.get("classes")
+    confidence = bounding_boxes.get("confidence", None)
```

Comment:
Was it just a bug that this was missing before? Or is it needed as a consequence of the change to the multi-class NMS?

I can't quite compute how it's related to the rest of the PR

Reply:
Its a bug, but  I had to fix it to get the PYCOCOCallback to operate correctly (it needs to match the other two)